data/items/items.xml

```diff
@@ -75820,5 +75820,294 @@ Granted by TibiaGoals.com"/>
+			</attribute>
+		</attribute>
+		<attribute key="script" value="moveevent">
+			<attribute key="level" value="250"/>
+			<attribute key="slot" value="armor"/>
+			<attribute key="vocation" value="Knight;true, Elite Knight"/>
+		</attribute>
+	</item>
+	<item id="44621" article="a" name="dauntless dragon scale armor">
+		<attribute key="primarytype" value="armors"/>
+		<attribute key="armor" value="18"/>
+		<attribute key="skillaxe" value="3"/>
+		<attribute key="skillsword" value="3"/>
+		<attribute key="skillclub" value="3"/>
+		<attribute key="absorbpercentphysical" value="8"/>
+		<attribute key="weight" value="15200"/>
+		<attribute key="imbuementslot" value="2">
+			<attribute key="life leech" value="3"/>
+			<attribute key="elemental protection death" value="3"/>
+			<attribute key="elemental protection earth" value="3"/>
+			<attribute key="elemental protection fire" value="3"/>
+			<attribute key="elemental protection ice" value="3"/>
+			<attribute key="elemental protection energy" value="3"/>
+			<attribute key="elemental protection holy" value="3"/>
+		</attribute>
+		<attribute key="augments" value="1">
+			<attribute key="front sweep" value="life leech">
+				<attribute key="value" value="4" />
+			</attribute>
+		</attribute>
+		<attribute key="script" value="moveevent">
+			<attribute key="level" value="300"/>
+			<attribute key="slot" value="armor"/>
+			<attribute key="vocation" value="Knight;true, Elite Knight"/>
+		</attribute>
+	</item>
+	<item id="44622" article="an" name="unerring dragon scale armor">
+		<attribute key="primarytype" value="armors"/>
+		<attribute key="armor" value="17"/>
+		<attribute key="skilldist" value="3"/>
+		<attribute key="holymagiclevelpoints" value="1"/>
+		<attribute key="absorbpercentenergy" value="10"/>
+		<attribute key="absorbpercentearth" value="-5"/>
+		<attribute key="weight" value="13500"/>
+		<attribute key="imbuementslot" value="2">
+			<attribute key="life leech" value="3"/>
+			<attribute key="elemental protection death" value="3"/>
+			<attribute key="elemental protection earth" value="3"/>
+			<attribute key="elemental protection fire" value="3"/>
+			<attribute key="elemental protection ice" value="3"/>
+			<attribute key="elemental protection energy" value="3"/>
+			<attribute key="elemental protection holy" value="3"/>
+		</attribute>
+		<attribute key="augments" value="1">
+			<attribute key="strong ethereal spear" value="life leech">
+				<attribute key="value" value="4" />
+			</attribute>
+		</attribute>
+		<attribute key="script" value="moveevent">
+			<attribute key="level" value="300"/>
+			<attribute key="slot" value="armor"/>
+			<attribute key="vocation" value="Paladin;true, Royal Paladin"/>
+		</attribute>
+	</item>
+	<item id="44623" article="an" name="arcane dragon robe">
+		<attribute key="primarytype" value="armors"/>
+		<attribute key="armor" value="15"/>
+		<attribute key="magiclevelpoints" value="3"/>
+		<attribute key="absorbpercentfire" value="-6"/>
```

Comment:
energymagiclevelpoints +1

Reply:
> energymagiclevelpoints +1

i cant take a print, but in tibia global have this energy magic level points +1

Reply:
I saw it on Tibia Wiki, Tibia Fandom does not have this information

Reply:
I'll add

Reply:
Fixed